asynchbase/README.md

```diff
@@ -21,9 +21,24 @@ This driver provides a YCSB workload binding for Apache HBase using an alternati
 
 ## Quickstart
 
-### 1. Setup Hbase
+### 1. Start a HBase Server
+You need to start a single node or a cluster to point the client at. Please see [Apache HBase Reference Guide](http://hbase.apache.org/book.html) for more details and instructions.
+
+### 2. Set up YCSB
+
+Download the [latest YCSB](https://github.com/brianfrankcooper/YCSB/releases/latest) file. Follow the instructions.
+
+### 3. Create a HBase table for testing
+
+For best results, use the pre-splitting strategy recommended in [HBASE-4163](https://issues.apache.org/jira/browse/HBASE-4163):
+
+```
+hbase(main):001:0> n_splits = 200 # HBase recommends (10 * number of regionservers)
```

Comment:
this is gross. we have the split algorithm classes... assuming they're now available on all release branches, we should promote those instead.

Reply:
I would love to have a simpler way to presplit. Can you make a PR with specifics?

Reply:
The class I was thinking of is `o.a.h.h.util.RegionSplitter$SplitAlgorithm`. `RegionSplitter` is marked as IA.Private, but `SplitAlgorithm` is consumed from public interfaces of the likes of `TableSnapshotInputFormat` and others. It does not seem to have an existing algorithm that is sympathetic to YCSB, but I suppose one could drop such an implementation into the class path of the target cluster...